src/index.ts

```diff
@@ -348,12 +351,115 @@ async function certificateForImpl<
     key: readFile(domainKeyPath),
     cert: readFile(domainCertPath)
   } as IReturnData<O>;
-  if (options.getCaBuffer) (ret as CaBuffer).ca = readFile(rootCACertPath);
-  if (options.getCaPath) (ret as CaPath).caPath = rootCACertPath;
+  if (options.getCaBuffer)
+    ((ret as unknown) as CaBuffer).ca = readFile(rootCACertPath);
+  if (options.getCaPath) ((ret as unknown) as CaPath).caPath = rootCACertPath;
 
   return ret;
 }
 
+/**
+ * Trust the certificate for a given hostname and port and add
+ * the returned cert to the local trust store.
+ * @param hostname - hostname of the remote machine
+ * @param port - port to connect the remote machine
+ * @param certPath - file path to store the cert
+ */
+async function trustCertsOnRemote(
```

Comment:
I think this function should be called something like `trustRemoteCerts` or `trustCerts` since it's not being run on the remote side, it's trusting the certs downloaded from the remote side on the local side.